git_remotes/remotes_in_git.md

```diff
@@ -0,0 +1,464 @@
+<!--
+
+author:   Julianna Pakstis
+email:    pakstisj@chop.edu
+version:  0.0.1
+language: en
+narrator: UK English Female
+title: Remotes in GitHub
+comment:  Learn how to share changes with others on the web. 
+long_description: Learn how to share changes to and gather changes from remote repositories in GitHub on the web. This module teaches how to share your work between repositories and how to pull the work of others into your repository. It reveals the power of git and Github for collaboration. 
+
+@learning_objectives  
+
+After completion of this module, learners will be able to:
+
+- Explain what remote repositories are and why they are useful 
+- Push to and pull from a remote repository
+
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/modules.css
+
+-->
+
+# Remotes in GitHub
+
+
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?** @long_description
+
+**Estimated time to completion:** 45 minutes-1 hour
+
+**Pre-requisites**
+
+List any skills and knowledge needed to do this module here. When available, include links to resources, especially other modules we've made (to show learners where this falls within our catalog).
+
+* A [GitHub](https://github.com/) account
+* (Preferred) Completion of git modules 1-6
+
+**Learning Objectives**
+
+@learning_objectives
+
+
+</div>
+
+## Lesson Preparation
+
+Open a web browser, like Chrome or Firefox and navigate to github.com.
+
+## Create a remote repository
+
+We will be creating a remote version of the `planets` repository we worked with in the previous lessons. We will then connect our remote version to our local version of the `planets` repository. 
+
+Log in to [GitHub](github.com), then click on the icon in the top right corner to create a new repository:
+
+![github-create-repo-01](./assets/media/remotes_step_01_images/github-create-repo-01.png)
+
+Name your repository “planets” and then click “Create Repository”.
+
+<div class = "warning">
+Since this repository will be connected to a local repository, it needs to be empty. Leave “Initialize this repository with a README” unchecked, and keep “None” as options for both “Add .gitignore” and “Add a license.” See the “GitHub License and README files” exercise below for a full explanation of why the repository needs to be empty.
+</div>
+
+![github-create-repo-02](./assets/media/remotes_step_01_images/github-create-repo-02.png)
+
+
+As soon as the repository is created, GitHub displays a page with a URL and some information on how to configure your local repository:
+
+![github-create-repo-03](./assets/media/remotes_step_01_images/github-create-repo-03.png)
+
+This effectively does the following on GitHub’s servers:
+
+```console
+$ mkdir planets
+$ cd planets
+$ git init
+```
+
+If you remember back to the earlier episode where we added and committed our earlier work on mars.txt, we had a diagram of the local repository which looked like this:
```

Comment:
Should I remove these SWC diagrams since they weren't used before? I could replace with a text description harkening back to the language ED used in the [Understanding the git workflow module](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/git_creation_and_tracking/git_creation_and_tracking/git_creation_and_tracking.md#16)